README.md.jinja

```diff
@@ -16,10 +16,14 @@
 [![codecov](https://codecov.io/gh/OCA/{{ repo_slug }}/branch/{{ odoo_version }}/graph/badge.svg)](https://codecov.io/gh/OCA/{{ repo_slug }})
 [![Translation Status](https://translation.odoo-community.org/widgets/{{ repo_slug }}-{{ odoo_version_slug(odoo_version) }}/-/svg-badge.svg)](https://translation.odoo-community.org/engage/{{ repo_slug }}-{{ odoo_version_slug(odoo_version) }}/?utm_source=widget)
 
+<!-- /!\ do not modify above this line -->
+
 # {{ repo_name }}
 
 {{ repo_description }}
 
+<!-- /!\ do not modify below this line -->
```

Comment:
Why two times the same text here?

Reply:
@pedrobaeza it's not the same ? there above and below

Reply:
Oh, yes, I read bad from mobile